cmd/disaster-recovery/disaster_recovery_test.py

```
@@ -0,0 +1,335 @@
+            import traceback
+            traceback.print_exc()
+            sys.exit(1)
+
+    def _extract_account_from_jwt(self):
+        """Extract account_id from JWT access token"""
+        try:
+            clean_token = ''.join(self.access_token.split())
+            parts = clean_token.split('.')
+
+            if len(parts) != 3:
+                print(f"[ERROR] Invalid JWT token format")
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)